wordpress-mobile/WordPress-iOS
# [Stats Refresh] DWMY Async loading and Ghost views - Part 2

Refs. #12147 

This PR continues the Load Blocks Independently implemented [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/12791).

In this PR I implemented the operations and the ghost views for these blocks only:
• _topAuthors_
• _topSearchTerms_
• _topCountries_
• _topVideos_
• _topFileDownloads_

## To test:
- Open Stats -> DWMY from a Site you never opened before (or from a fresh install)
- You shouldn't see the NRV
- You should be able to load all the cards
- Disable the feature flag and make sure everything works as before

Update release notes:
